### DISCUSSION TITLE
When Building Docker Image for infinity is there a flag to implement queueing?

### DISCUSSION THREAD

**Discussion by janakm-senslyze** [michaelfeil/infinity#305] [Q&A]
I have used BAAI/bge-large-en-v1.5 embeddings and hosted it using docker on my own linux server.

When i embed 3 pdf documents and 4 cpu cores it works fine.
BUT, when I embed 10 pdf documents with 4 cpu cores then it dosent finish. (the documents are not very large around 20 pages each).
After half an hour i gave up and cancelled the operation.

Is it an issue of bge embeddings?
and is queueing already implemented inside code for infinity embeddings?
Please help.

**Comment by michaelfeil** [comment]
There’s a queue inside. Are you sending http requests? What its their response code? Do they time out etc?
